lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -41,7 +41,7 @@ def define
           going_or_already_abroad == 'already_abroad'
         end
 
-        calculate :already_abroad_text_two do |response|
+        calculate :already_abroad_text_two do |_response|
```

Comment:
I did mean to remove it, but forgot. Well spotted!

Addressed in 28d34a9494f26218033a9de5903c3585942ca98b.